Artifacts/linux-syscoin/build_syscoin.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e 
+
+date
+ps axjf
+
+
+if [ $1 = 'From_Source' ]; then
+#################################################################
+# Build Syscoin from source                                     #
+#################################################################
+NPROC=$(nproc)
+echo "nproc: $NPROC"
+#################################################################
+# Install all necessary packages for building Syscoin           #
+#################################################################
+sudo apt-get -y install git build-essential libtool autotools-dev autoconf pkg-config libssl-dev libevent-dev bsdmainutils libboost-all-dev libminiupnpc-dev libzmq3-dev
```

Comment:
Not all Linux distros have apt-get. In other artifacts we've added conditional support for apt-get or yum. Here's an example: https://github.com/Azure/azure-devtestlab/blob/master/Artifacts/linux-install-mongodb/linux_install_mongodb_v1.sh